### ISSUE TITLE
[BUG] Calendar events with the new fields not in DEV, TEST, or PROD

### ISSUE THREAD

**Issue by sandeep-ps** [rokwire/events-manager#928]
**Describe the bug**
Calendar events (i.e., from WebTools) with the new fields (isInPerson, virtualEventUrl) are not in DEV, TEST, or PROD at present. Only some test events are present.

**To Reproduce**
1. Run the following command in at least DEV and TEST:
```
curl --location --request GET 'https://api-dev.rokwire.illinois.edu/events?startDate.gte=2022-07-21T09:07:09&isInPerson=true' \
--header 'ROKWIRE-API-KEY: <ROKWIRE API KEY>'
```
2. See that the events returned are test events manually uploaded and not events from Web Tools.
3. Go to https://xml.calendars.illinois.edu/eventXML16/6991.xml from the VPN
4. See that there are many events from WebTools with the above fields.

**Expected behavior**
The new calendar data ingestion should happen correctly, and the event with new fields should be retrievable through the Events BB.

**Screenshots**

**Desktop (please complete the following information):**
 - Version 3.3.0

**Additional context**

**Comment by bingzhang** [comment]
@sandeep-ps i will take a look at it.

**Comment by bingzhang** [comment]
closed. fixed in 3.3.1.

**Comment by bingzhang** [comment]
https://github.com/rokwire/events-manager/pull/929
